pkg/multicluster/register/helpers.go

```diff
@@ -248,27 +226,6 @@ func defaultRegistrant(cfg *rest.Config, apiServerAddress string) (ClusterRegist
 	return registrant, nil
 }
 
-// Attempts to load a Client KubeConfig from a default list of sources.
-func getClientConfigWithContext(masterURL, kubeCfgPath, context string) (clientcmd.ClientConfig, error) {
```

Comment:
can we expose this as a util function? seems like all the consuming projects would need to re-implement otherwise

Reply:
This is already [available here](https://github.com/solo-io/skv2/blob/b3439f5151a08654a765bce3111d2fdcdfa5009e/pkg/multicluster/kubeconfig/loader.go#L23)